crates/bevy_ecs/src/bundle.rs

```diff
@@ -8,13 +8,33 @@ use crate::{
 use bevy_ecs_macros::all_tuples;
 use std::{any::TypeId, collections::HashMap};
 
-/// An ordered collection of components
-///
-/// See [Bundle]
-/// # Safety
-/// [Bundle::type_info] must return the TypeInfo for each component type in the bundle, in the _exact_
-/// order that [Bundle::get_components] is called.
-/// [Bundle::from_components] must call `func` exactly once for each [TypeInfo] returned by [Bundle::type_info]
+/** An ordered collection of components, commonly used for spawning entities, and adding and removing components in bulk.
```

Comment:
We use `///`-style doc comments everywhere else. Can we do so here as well? Most other rust projects ive seen use that style too (including rustc)

Reply:
Can do: I was just being lazy with it for the code example.